.github/workflows/uploadFirrtlReleaseArtifacts.yml

```diff
@@ -146,31 +156,62 @@ jobs:
           file install/*
           file install/bin/*
 
+      # Specify bash for the Windows runner to work
       - name: Name Install Directory
         id: name_dir
+        shell: bash
         run: |
           BASE=$(git describe --tag)
           SANITIZED=$(echo -n $BASE | tr '/' '-')
           echo "value=$SANITIZED" >> "$GITHUB_OUTPUT"
 
-      - name: Package Binaries
+      - name: Name Archive
+        id: name_archive
+        shell: bash
+        run: |
+          NAME=firrtl-bin-${{ matrix.os }}-${{ matrix.arch }}.${{ matrix.archive }}
```

Comment:
This was discussed here: https://github.com/llvm/circt/pull/4887#discussion_r1149896146

I ended up going with @darthscsi's suggestion:
> I might go with firrtl-bin based on the use case rather than the executable name.

 I still don't really have a strong opinion, firrtl is the use case, firtool is the name of the binary. In theory we could add more things (eg. a shared library that Chisel links against directly, or `arcsim` as a default simulator for Chisel) so there is some niceness is keeping the name based on the use case, but I agree at the moment it feels a bit weird.

Reply:
Interesting, okay thanks! :+1: